deepfence_bootstrapper/router/secret_scanner.go

```diff
@@ -111,3 +111,26 @@ func GetSecretScannerJobCount() int32 {
 	}
 	return jobReport.RunningJobs
 }
+
+func StopSecretScan(req ctl.StopSecretScanRequest) error {
+	fmt.Printf("Stop Secret Scan : %v\n", req)
+	conn, err := grpc.Dial("unix://"+ebpfSocketPath, grpc.WithAuthority("dummy"),
+		grpc.WithInsecure())
```

Comment:
Not for this PR but we probably need to revamp so that we only have 1 client.
With that said, maybe gRPC client is already clever enough and reuse connections